napari/_vispy/_tests/test_vispy_scale_bar_visual.py

```diff
@@ -6,3 +6,10 @@ def test_scale_bar_instantiation(make_napari_viewer):
     viewer = make_napari_viewer()
     model = ScaleBarOverlay()
     VispyScaleBarOverlay(overlay=model, viewer=viewer)
+
+
+def test_scale_bar_fixed_width(make_napari_viewer):
+    viewer = make_napari_viewer()
+    model = ScaleBarOverlay(fixed_width=50)
+    vispy_scale_bar = VispyScaleBarOverlay(overlay=model, viewer=viewer)
+    assert vispy_scale_bar.overlay.fixed_width == 50
```

Comment:
spawning a full viewer for this is a bit much; I think it's better to just grow the test above a bit and do set/get to check that the connection works!